lib/identity_cache/configuration_dsl.rb

```diff
@@ -279,7 +279,7 @@ def attribute_dynamic_fetcher(attribute, fields, values) #:nodoc:
       end
 
       def add_parent_expiry_hook(options)
-        child_class = options[:association_class]
+        child_class = options[:association_class].klass
```

Comment:
Thanks for comments, @dylanahsmith – do You have an idea how to deal with this one? here child_class must be known at class_eval time :/